### PR TITLE
Add Carol Willing and Steve Silvester to the steering council.

### PR DESCRIPTION
**Pull request by Carreau**
cc @fperez , @willingc and @blink1073

I will merge soon.

### CODE REVIEW

File: people.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Fernando Perez is the Benevolent Dictator for Life (BDFL).
 - Sylvain Corlay
 - Thomas Kluyver
 - Peter Parente
+- Carol Willing
+- Steve Silvester
```
Comment (blink1073) [inline people.md]: Should our names be added to the institutional partners as well?

Comment (Carreau) [inline people.md]: Oops, Sorry.

**Comment by blink1073** [comment]
Looks great, thank you!

**Comment by Carreau** [comment]
Both should be fixed. @willingc I've put you under CalPoly , or do you prefer willingconsulting.com

File: people.md [hunk 2]
```diff
@@ -33,8 +35,8 @@ Fernando Perez is the Benevolent Dictator for Life (BDFL).
 
 Institutional Council members are noted with each institution.
 
-- [Bloomberg](http://www.bloomberg.com/) (Jason Grout, Sylvain Corlay)
-- [Cal Poly](http://www.calpoly.edu/) (Brian Granger)
+- [Bloomberg](http://www.bloomberg.com/) (Jason Grout, Sylvain Corlay, Steven Silvester)
```
Comment (blink1073) [inline people.md]: I missed this early, but my direct affiliation is with Continuum.

**Comment by willingc** [comment]
@carreau Cal Poly is great. Thanks for doing this. :cake:

Congrats to the amazing @blink1073 too :tada:

**Comment by blink1073** [comment]
Congratulations to you as well @willingc!

File: people.md (hunk 2 above)
Comment (Carreau) [inline people.md]: Oh gosh I'm not awake this morning should be fixed.,

**Comment by Carreau** [comment]
And I fix the team to match the steering council, a few people were missing.

**Comment by rgbkrk** [comment]
_hovers over merge button_

**Comment by Carreau** [comment]
> hovers over merge button

What are you waiting for.

**Comment by willingc** [comment]
Thanks @SylvainCorlay 🌞

**Comment by Carreau** [comment]
Sweet, can [this](https://github.com/jupyter/jupyter.github.io/pull/158) be merged as well ?

**Pull request merged**